### PR TITLE
feat: schema/data toggle

### PR DESCRIPTION
**Pull request by quesebifurcan**

### CODE REVIEW

File: aether-ui/aether/ui/assets/apps/components/NavBar.jsx [hunk 1]
```diff
@@ -34,7 +34,7 @@ class NavBar extends Component {
         }
 
         <div data-qa='navbar-user' className='top-nav-user'>
-          <span class='user-name'>
+          <span className='user-name'>
```
Comment (obdulia-losantos) [inline aether-ui/aether/ui/assets/apps/components/NavBar.jsx:37]: good catch!

File: aether-ui/aether/ui/assets/apps/pipeline/Pipeline.jsx [hunk 2]
```diff
@@ -58,7 +58,7 @@ class Pipeline extends Component {
     return (
       <div className={'pipelines-container show-pipeline'}>
         <NavBar showBreadcrumb>
-          <div classname='breadcrumb-links'>
+          <div className='breadcrumb-links'>
```
Comment (obdulia-losantos) [inline aether-ui/aether/ui/assets/apps/pipeline/Pipeline.jsx:61]: good catch! again!

**Comment by obdulia-losantos** [comment]
LGTM!

One concern I have is that if the user updates the input data without changing the schema but just given meaningful values to the properties this makes the AVRO schema to be regenerated and the other way around... is there a way to detect those cases?

**Review by lordmallam — APPROVED** [review]
Looks good

**Comment by quesebifurcan** [comment]
@obdulia-losantos good point -- me, Adam and Sabeen discussed this offline on Wednesday. It will be addressed in a future PR. The idea was that, when editing an Avro schema generated from a data sample, a gentle reminder will appear above the text area, notifying the user that committing these changes will result in in the sample data being generated anew (if no longer compatible with the edited schema).

**Pull request merged**